magneticod/magneticod/dht.py

```diff
@@ -261,3 +245,23 @@ def __calculate_token(self, addr: NodeAddress, info_hash: InfoHash):
     @staticmethod
     def __random_bytes(n: int) -> bytes:
         return random.getrandbits(n * 8).to_bytes(n, "big")
+
+    def __build_request_find_node(self, id: bytes) -> bytes:
```

Comment:
* This method should be renamed into `__build_FIND_NODE_query` for consistency.
* `ìd` argument should be renamed as `id_` to prevent clash with the built-in function `id()`.